src/features/deposit/components/DepositForm/index.tsx

```diff
@@ -323,3 +334,11 @@ function getBlockchainsOptions(): Record<
 function shortenTxHash(txHash: string) {
   return `${txHash.slice(0, 5)}...${txHash.slice(-5)}`
 }
+
+function getDefaultBlockchainOption(token: SwappableToken) {
+  if (token && !isUnifiedToken(token)) {
+    const key = `${token.chainName}:${token.chainId}`
+    return getBlockchainsOptions()[key]
+  }
+  return undefined
```

Comment:
If it is `BaseToken` then the value should be `{token.chainName}:${token.chainId}`, because BaseToken always has blockchain info.